### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ethanrossblack**
## Explain changes:

Add a bunch of the documentation and queries to the BE readme for more centralized/production documentation

### What does this PR do?
- [ ] Feature
- [ ] Bug fix
- [ ] Refactor
- [ ] Styling
- [x] Other: Documentation

### Necessary checkmarks:
- [ ] All Tests are Passing
- [ ] The code will run locally

### Testing Changes
- [ ] No Tests have been changed/added
- [ ] Some Tests have been changed/added
- [ ] All of the Tests have been changed(Please explain)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,190 @@
...
+      "latitude": 35.653564198354196,
+      "longitude": -97.19134672640317,
+      "shareAddress": false,
+      "sharePhone": true,
+      "shareEmail": true,
+      "aidRequests": [
+        {
+          "id": "2000",
+          "organizationId": 1,
+          "aidType": null,
```
Comment (DavisWeimer) [inline README.md:140]: This should be fixed in the db now after we merge #28! So it won't return `nil` 
I guess you can just update these with a random language

File: README.md [hunk 2]
```diff
@@ -1,24 +1,190 @@
...
+          "language": "Arabic",
+          "description": "MyText",
+          "status": null
```
Comment (DavisWeimer) [inline README.md:143]: Same with `status`, dang ol' misbehaving `enums`!

File: README.md [hunk 3]
```diff
@@ -1,24 +1,190 @@
-# README
+# Refugee Aid
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Refugee Aid is an app that connects citizens in the United States with refugee aid organizations and camps.
 
-Things you may want to cover:
+## :clipboard: Table of Contents
```
Comment (DavisWeimer) [inline README.md:5]: GREAT! Thank you for tackling this, formating is gonna look dope

File: README.md [hunk 4]
```diff
@@ -1,24 +1,190 @@
...
+</td>
+<td align="center">
+Column 2
+</td>
+</tr>
+</table>
+
+## :people_holding_hands: Authors
+
+**Wutz** is a student project built in September, 2023 for the Backend Program of the [Turing School of Software and Design](https://turing.edu/).
```
Comment (algibson1) [inline README.md:181]: Looks like a comment was left over after copy-pasting!

**Review by algibson1 — APPROVED** [review]
Approving

**Pull request merged**